modules/material-management/test/keyring.test.ts

```diff
@@ -46,7 +45,7 @@ describe('Keyring', () => {
   it('onEncrypt calls _onEncrypt', async () => {
     const suite = new NodeAlgorithmSuite(AlgorithmSuiteIdentifier.ALG_AES128_GCM_IV12_TAG16)
     const { keyLengthBytes } = suite
-    const m = new NodeEncryptionMaterial(suite)
+    const m = new NodeEncryptionMaterial(suite, {})
```

Comment:
Nitpick: Would prefer a more descriptive name here.

Reply:
What are you looking for?  Defining the variable as opposed to just passing an object lateral?

Reply:
Oh, sorry, I meant `m` not `{}`. Not a big deal.